File: Javascript/ninety-nine-bottles-js/beer.js
bottlesOfBeer = 99;

while (bottlesOfBeer > 1) {
  if (bottlesOfBeer % 10 == 0) {
    punctuation = `!`;
  } else {
    punctuation = `.`; }
 console.log(`\n${bottlesOfBeer} bottles of beer on the wall, ${bottlesOfBeer} bottles of beer${punctuation}`);
 bottlesOfBeer--;
 console.log(`Take one down and pass it around, ${bottlesOfBeer} bottles of beer on the wall.`);
}

console.log(`\n${bottlesOfBeer} bottle of beer on the wall, ${bottlesOfBeer} bottle of beer.\nTake one down and pass it around, no more bottles of beer on the wall.`);
console.log(`\nNo more bottles of beer on the wall, no more bottles of beer!\nGo to the store and buy some more, then find a rehab because you have a serious problem.`);

File: Ruby/ninety-nine-bottles/beer.rb
bottles_of_beer = 99

while bottles_of_beer > 1 do
  print "\n#{bottles_of_beer} bottles of beer on the wall, #{bottles_of_beer} bottles of beer"

  if bottles_of_beer % 10 == 0
    print "!"
  else
    print "."
  end

  bottles_of_beer -= 1
  puts "\nTake one down and pass it around, #{bottles_of_beer} bottles of beer on the wall."
end

puts "\n#{bottles_of_beer} bottle of beer on the wall, #{bottles_of_beer} bottle of beer.\n" +
     "Take one down and pass it around, no more bottles of beer on the wall."

puts "\nNo more bottles of beer on the wall, no more bottles of beer!\n" +
     "Go to the store and buy some more, then find a rehab because you have a serious problem."
